src/ResourceManager/RedisCache/Commands.RedisCache/Models/RedisCacheAttributesWithAccessKeys.cs

```diff
@@ -18,7 +18,7 @@ namespace Microsoft.Azure.Commands.RedisCache.Models
 
     public class RedisCacheAttributesWithAccessKeys : RedisCacheAttributes
     {
-        public RedisCacheAttributesWithAccessKeys(RedisResourceWithAccessKey cache, string resourceGroupName)
```

Comment:
@SiddharthChatrolaMs customers are still able to create this C# object in PowerShell, so it would be a breaking change if they are using this constructor